learn4haskell.cabal

```diff
@@ -56,3 +56,35 @@ test-suite learn4haskell-test
   ghc-options:         -threaded
                        -rtsopts
                        -with-rtsopts=-N
+
+test-suite doctest-chapter1
+  import:              common-options
+  type:                exitcode-stdio-1.0
+  hs-source-dirs:      test
+  main-is:             DoctestChapter1.hs
+  build-depends:       doctest ^>= 0.17
+  ghc-options:         -threaded
```

Comment:
Can we have a separate common stanza for doctests and move `hs-source-dirs`, `build-depends` and `ghc-options` in it?